modules/create-data-object/pkg/constants/constants.go

```diff
@@ -0,0 +1,31 @@
+package constants
+
+import "time"
+
+// Exit codes
+const (
+	InvalidCLIInputExitCode    = -1
+	DataObjectCreatorErrorCode = 1
```

Comment:
2 got missing? :D

Reply:
The structure was borrowed from another task :D I changed it